ember/app/controllers/map/moderations/index.js

```diff
@@ -14,6 +14,11 @@ export default class extends ModerationController {
                .filter(moderation => !moderation.get('approved'));
   }
 
+  @computed('developments.[]')
+  get filteredFlags() {
+    return this.store.findAll('development');
```

Comment:
This will be a hefty lift. We can hijack the filtering endpoint to only return what we need. Try:

```js
return this.store.query('development', { trunc: true, filter: { flag: true } });
```
The `trunc` param isn't necessary to do what we want but it's a performance optimization. We can use the truncated form of the filtered results since you are only displaying `development.name`. In order to use the `trunc`, remember to change the ActiveModel serializer as well.

Reply:
This almost worked except it didn't actually filter. I think the issue might be that Rails isn't doing anything with the filter param, though I thought the filter was either included by magic or an ember side thing.